test-requirements.txt

```diff
@@ -2,3 +2,4 @@ black==19.10b0
 flake8
 pytest
 yamllint
+pyarrow
```

Comment:
This bit might be problematic with `apache_beam` as it will downgrade pyarrow to 0.17.0 or something.  Is there a reason (I don't recall) why we needed this specifically for test?

Reply:
According to git, it looks like you added that line 3 days ago.  In so far as tests need armory installed before they will run, this is probably better put in no-docker-requirements.txt along with apache-beam (and pin the version).

Our code makes no direct reference to pyarrow.

Reply:
Perfect, that is my bad, I will remove.